src/view/utils.py

```diff
@@ -0,0 +1,134 @@
+from PyQt5 import QtWidgets
+import pandas as pd
+import numpy as np
+from src.view import ui
+
+
+def is_leaf(dic, d):
```

Comment:
Some documentation to explain at least what is `dic` and `d`

Reply:
done!